scripts/content.js

```diff
@@ -0,0 +1,8 @@
+chrome.runtime.onMessage.addListener((obj, sender, response) => {
+  const { onMedium, error } = obj;
+  if (response && onMedium) {
+    localStorage.clear();
```

Comment:
I'm clearing local storage just in case Medium is checking to see if the number of articles read doesn't match the cookies (to identify programs like this).